sentry_streams/sentry_streams/adapters/loader.py

```diff
@@ -28,6 +30,9 @@ def load_adapter(adapter_type: str, config: PipelineConfig) -> StreamAdapter[Str
     #TODO: Actually move out Flink otherwise everything stated above makes
     # no sense.
     """
+    if segment_id is not None:
+        config = {"env": config["env"], **config["pipeline"]["segments"][segment_id]}
+
```

Comment:
is segment id a relevant field for an adapter like Flink ? 
It is alright here. Though food for thoughts